WSSiOS/WSSiOS/Resource/Constants/URLs/URLs.swift

```diff
@@ -65,4 +65,11 @@ enum URLs {
     enum Feed {
         static let getFeeds = "/feeds"
     }
+    
+    enum MyPage {
+        enum Block {
+            static let blocks = "/blocks"
+            static let userBlocks = "/blocks/{blockId}"
```

Comment:
W2
```suggestion
            static func userBlocks(blockId: Int) -> String {
            return "/blocks/\(blockId)"
        }
```

요렇게 쓰면 호출할 때 쫌 더 깔꼼!